addons/mail/static/tests/helpers/mock_models.js

```diff
@@ -68,6 +68,7 @@ class MockModels {
             'mail.channel': {
                 fields: {
                     channel_type: { string: "Channel Type", type: "selection", default: 'channel' },
+                    channel_partner_ids: { string: "Channel Partner Ids", type: 'many2many', relation: 'res.partner' },
```

Comment:
add comment this is equivalent to members but needed due to some RPC giving this field in domain